components/ViewArticlePage/DetailsPanel.tsx

```diff
@@ -0,0 +1,48 @@
+import { Paper, Typography } from '@mui/material'
+
+const DetailSegment = ({ title, desc }: { title: string; desc: string }) => {
+  return (
+    <>
+      <Typography fontSize={'12px'} fontWeight={300}>
+        {title}
+      </Typography>
+      <Typography fontSize={'16px'} fontWeight={400} marginBottom={'2px'}>
+        {desc}
+      </Typography>
+    </>
+  )
+}
+
+const DetailsPanel = () => {
+  return (
+    <Paper
+      elevation={2}
+      sx={{
+        height: 410,
+        borderRadius: 3,
+        marginBottom: '20px',
+        padding: '24px',
+      }}
+    >
+      <Typography fontSize={'21px'} fontWeight={700} marginBottom="10px">
```

Comment:
just a nit remark, when passing strings, we dont need to wrap it in a `{}`. so this line could just be `fontSize="12px"`. but u don't have to change it, just to let u know